http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -0,0 +1,132 @@
+      applicationInsightsComponent <- azureApplicationInsightsService.getApplicationInsights(
+        params.landingZoneResources.applicationInsightsName,
+        params.cloudContext
+      )
+
+      // Storage container is required for Cromwell app
+      storageContainer <- F.fromOption(
+        params.storageContainer,
+        AppCreationException("Storage container required for Cromwell Runner app", Some(ctx.traceId))
+      )
+
+      // Databases required for Cromwell App
+      dbNames <- F.fromOption(toCromwellRunnerAppDatabaseNames(params.databaseNames),
+                              AppCreationException("Database names required for Cromwell Runner app", Some(ctx.traceId))
+      )
+
+      // Postgres server required for Cromwell App
+      postgresServer <- F.fromOption(
+        params.landingZoneResources.postgresServer,
+        AppCreationException("Postgres server required for Cromwell Runner app", Some(ctx.traceId))
+      )
+
+      // Get the pet userToken
+      tokenOpt <- samDao.getCachedArbitraryPetAccessToken(params.app.auditInfo.creator)
+      userToken <- F.fromOption(
+        tokenOpt,
+        AppCreationException(s"Pet not found for user ${params.app.auditInfo.creator}", Some(ctx.traceId))
+      )
+
+      values = List(
+        // azure resources configs
+        raw"config.resourceGroup=${params.cloudContext.managedResourceGroupName.value}",
+        raw"config.batchAccountKey=${batchAccount.getKeys().primary}",
+        raw"config.batchAccountName=${params.landingZoneResources.batchAccountName.value}",
+        raw"config.batchNodesSubnetId=${params.landingZoneResources.batchNodesSubnetName.value}",
+        raw"config.drsUrl=${drsConfig.url}",
+        raw"config.landingZoneId=${params.landingZoneResources.landingZoneId}",
+        raw"config.subscriptionId=${params.cloudContext.subscriptionId.value}",
+        raw"config.region=${params.landingZoneResources.region}",
+        raw"config.applicationInsightsConnectionString=${applicationInsightsComponent.connectionString()}",
+
+        // relay configs
+        raw"relay.path=${params.relayPath.renderString}",
+
+        // persistence configs
+        raw"persistence.storageResourceGroup=${params.cloudContext.managedResourceGroupName.value}",
+        raw"persistence.storageAccount=${params.landingZoneResources.storageAccountName.value}",
+        raw"persistence.blobContainer=${storageContainer.name.value}",
+        raw"persistence.leoAppInstanceName=${params.app.appName.value}",
+        raw"persistence.workspaceManager.url=${params.config.wsmConfig.uri.renderString}",
+        raw"persistence.workspaceManager.workspaceId=${params.workspaceId.value}",
+        raw"persistence.workspaceManager.containerResourceId=${storageContainer.resourceId.value.toString}",
+
+        // identity configs
+        raw"workloadIdentity.serviceAccountName=${params.ksaName.value}",
+        raw"identity.name=${params.managedIdentityName.value}",
```

Comment:
`identity.name` is still required -- see https://github.com/broadinstitute/terra-helmfile/commit/010a274595ed155e945adc3099e55385bb2b0bae